src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -95,6 +95,32 @@ private CSharpCommandLineArguments FullParse(string commandLine, string baseDire
             return CSharpCommandLineParser.Default.Parse(args, baseDirectory, sdkDirectory, additionalReferenceDirectories);
         }
 
+        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.TestHasWindowsPaths)]
```

Comment:
Where is the windows path here?

Reply:
There technically isn't one, but I'm running an exe, which only works on Windows desktop and needed a reason.